test/views/coronavirus_local_restrictions/show_html_test.rb

```diff
@@ -0,0 +1,38 @@
+require "test_helper"
+require_relative "../../../test/support/coronavirus_helper"
+
+module CoronavirusLocalRestrictions
+  class ShowHtmlTest < ActionView::TestCase
+    include CoronavirusLocalRestrictionsHelpers
+    helper Rails.application.helpers
```

Comment:
I couldn't work out how this could happen automatically 😢 . It looks like we'll need this on any view tests that use helpers outside of the default ones that Rails has. 

It's annoying as auto loading of all helpers works fine in RSpec.